Specs/Scene/PointPrimitiveCollectionSpec.js

```diff
@@ -0,0 +1,921 @@
+            },
+            outlineWidth : 4.0,
+            scaleByDistance : new NearFarScalar(1.0, 3.0, 1.0e6, 0.0),
+            translucencyByDistance : new NearFarScalar(1.0, 1.0, 1.0e6, 0.0),
+            id : 'id'
+        });
+
+        expect(b.show).toEqual(false);
+        expect(b.position).toEqual(new Cartesian3(1.0, 2.0, 3.0));
+        expect(b.pixelSize).toEqual(2.0);
+        expect(b.color.red).toEqual(0.1);
+        expect(b.color.green).toEqual(0.2);
+        expect(b.color.blue).toEqual(0.3);
+        expect(b.color.alpha).toEqual(0.4);
+        expect(b.outlineColor.red).toEqual(0.5);
+        expect(b.outlineColor.green).toEqual(0.6);
+        expect(b.outlineColor.blue).toEqual(0.7);
+        expect(b.outlineColor.alpha).toEqual(0.8);
+        expect(b.outlineWidth).toEqual(4.0);
+        expect(b.scaleByDistance).toEqual(new NearFarScalar(1.0, 3.0, 1.0e6, 0.0));
+        expect(b.translucencyByDistance).toEqual(new NearFarScalar(1.0, 1.0, 1.0e6, 0.0));
+        expect(b.id).toEqual('id');
+    });
+
+    it('set pointPrimitive properties', function() {
+        var b = pointPrimitives.add();
+        b.show = false;
+        b.position = new Cartesian3(1.0, 2.0, 3.0);
+        b.pixelSize = 2.0;
+        b.color = new Color(0.1, 0.2, 0.3, 0.4);
+        b.outlineColor = new Color(0.5, 0.6, 0.7, 0.8);
+        b.outlineWidth = 4.0;
+        b.scaleByDistance = new NearFarScalar(1.0e6, 3.0, 1.0e8, 0.0);
+        b.translucencyByDistance = new NearFarScalar(1.0e6, 1.0, 1.0e8, 0.0);
+
+        expect(b.show).toEqual(false);
+        expect(b.position).toEqual(new Cartesian3(1.0, 2.0, 3.0));
+        expect(b.pixelSize).toEqual(2.0);
+        expect(b.color.red).toEqual(0.1);
+        expect(b.color.green).toEqual(0.2);
+        expect(b.color.blue).toEqual(0.3);
+        expect(b.color.alpha).toEqual(0.4);
+        expect(b.outlineColor.red).toEqual(0.5);
+        expect(b.outlineColor.green).toEqual(0.6);
+        expect(b.outlineColor.blue).toEqual(0.7);
+        expect(b.outlineColor.alpha).toEqual(0.8);
+        expect(b.outlineWidth).toEqual(4.0);
+        expect(b.scaleByDistance).toEqual(new NearFarScalar(1.0e6, 3.0, 1.0e8, 0.0));
+        expect(b.translucencyByDistance).toEqual(new NearFarScalar(1.0e6, 1.0, 1.0e8, 0.0));
+    });
+
+    it('disable pointPrimitive scaleByDistance', function() {
+        var b = pointPrimitives.add({
+            scaleByDistance : new NearFarScalar(1.0, 3.0, 1.0e6, 0.0)
+        });
+        b.scaleByDistance = undefined;
+        expect(b.scaleByDistance).not.toBeDefined();
+    });
+
+    it('disable pointPrimitive translucencyByDistance', function() {
+        var b = pointPrimitives.add({
+            translucencyByDistance : new NearFarScalar(1.0, 1.0, 1.0e6, 0.0)
+        });
+        b.translucencyByDistance = undefined;
+        expect(b.translucencyByDistance).not.toBeDefined();
+    });
+
+    it('render pointPrimitive with scaleByDistance', function() {
```

Comment:
Oh my, these are based on very old tests that execute commands directly instead of using `Scene`.

@emackey want to be a hero...a real hero?  It would be awesome to change these and the billboard and label tests to use the `Scene` instead similar to [ModelSpec](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Scene/ModelSpec.js)

See #1259.